demisto_sdk/commands/validate/file_validator.py

```diff
@@ -366,7 +366,7 @@ def validate_modified_files(self, modified_files):  # noqa: C901
 
             elif checked_type(file_path, JSON_ALL_INCIDENT_TYPES_REGEXES):
                 incident_type_validator = IncidentTypeValidator(structure_validator)
-                if not incident_type_validator.is_valid_incident_type():
+                if not incident_type_validator.is_valid_incident_type(validate_rn=True):
```

Comment:
why is that one different? why not `not file_type`?

Reply:
in `modified_files` it should always be true
in `added_files` it is `not file_type`

Reply:
right, my bad. sorry.